src/rdkafka.c

```diff
@@ -396,7 +396,7 @@ static const struct rd_kafka_err_desc rd_kafka_err_descs[] = {
 	_ERR_DESC(RD_KAFKA_RESP_ERR__MSG_TIMED_OUT,
 		  "Local: Message timed out"),
 	_ERR_DESC(RD_KAFKA_RESP_ERR__PARTITION_EOF,
-		  "Broker: No more messages"),
+		  "Local: No more messages"),
```

Comment:
Yeah, the reason this one differs is to help the user understand that there are no more messages on the broker, and not in any local queue.
So while the error string is technically incorrect I don't fixing it makes it easier on the user.

Reply:
Could it say `"Local: Broker has no more messages"` or `"Local: No more messages at broker"`?

Reply:
Good point :+1: